From 2f87cca2e129471f57df0c3a8a3adc0c73a4811c Mon Sep 17 00:00:00 2001
From: Patrick Hodoul <patrickhodoul@gmail.com>
Date: Tue, 24 Mar 2020 15:26:17 -0400
Subject: [PATCH] Adsk Contrib - Fix macos issues (#973)

Signed-off-by: Patrick Hodoul <Patrick.Hodoul@autodesk.com>
---
 src/core/ImageDesc.cpp        | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

--- a/src/core/ImageDesc.cpp
+++ b/src/core/ImageDesc.cpp
@@ -57,8 +57,8 @@ OCIO_NAMESPACE_ENTER
             os << "gData=" << planarImg->getGData() << ", ";
             os << "bData=" << planarImg->getBData() << ", ";
             os << "aData=" << planarImg->getAData() << ", ";
-            os << "width=" << packedImg->getWidth() << ", ";
-            os << "height=" << packedImg->getHeight() << ", ";
+            os << "width=" << planarImg->getWidth() << ", ";
+            os << "height=" << planarImg->getHeight() << ", ";
             os << "yStrideBytes=" << planarImg->getYStrideBytes() << "";
             os << ">";
         }
