From: "Matteo F. Vescovi" <mfv@debian.org>
Date: Thu, 18 Jan 2018 22:58:51 +0100
Subject: Fix_FTBFS_on_non-Linux_platforms

---
 src/core/Platform.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/Platform.h b/src/core/Platform.h
index c1796e8..2155252 100644
--- a/src/core/Platform.h
+++ b/src/core/Platform.h
@@ -85,7 +85,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.
 // assume linux/unix/posix
 
 #include <stdlib.h>
-#if !defined(__FreeBSD__)
+#if !defined(__FreeBSD__) || !defined(__FreeBSD_kernel__)
 #include <alloca.h>
 #endif
 #include <string.h>
